### PR TITLE
Add node/npm to compose file

### PR DESCRIPTION
**Pull request by pscheit**
Does npm install and npm run build once `docker compose up` is used.

Can put this to https://docs.docker.com/compose/profiles/ if needed. But dont think so.

- `version:` is not read from compose v2 anymore (always uses the latest)
- anonymous volumes can be now created without a top level entry in volume

### CODE REVIEW

**Comment by chr-hertel** [comment]
i usually prefer not to mount the working directory to a container and work with symfony cli and nvm to have the runtimes on host, but that's opinionated i guess - but feels a bit inconsistent to me now to have a node container, but no php one.

what about adding the override to ignore now?

**Comment by pscheit** [comment]
You mean adding the profile, so that its not started automatically?

If this doesn provide value to other Docker users I am fine to not merge this

**Comment by chr-hertel** [comment]
No, the changes make sense and adding a profile to the node one or adding `docker-compose.override.yml` to `.gitignore` or both would be totally compatible with my way to use the docker setup

**Comment by pscheit** [comment]
yeah, this is what I would prefer over profile as well. 

Would add docker-compose.node.yaml then

File: README.md [hunk 1]
```diff
@@ -53,7 +54,7 @@ These steps are provided for development purposes only.
    bin/console doctrine:schema:create
    ```
 7. Run a CRON job `bin/console packagist:run-workers` to make sure packages update.
-8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files.
+8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files. When using Docker run `docker-compose run node npm run dev` to watch css/js files.
```
Comment (Seldaek) [inline README.md:57]: Shouldn't this be a space here? I know nothing but seems inconsistent :D

```suggestion
8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files. When using Docker run `docker compose run node npm run dev` to watch css/js files.
```

Comment (pscheit) [reply README.md:57]: oh yeah totally. Bad finger habit!

Comment (pscheit) [reply README.md:57]: docker-compose is legacy v1 where as docker compose (compose as a plugin to docker) is correct

**Comment by Seldaek** [comment]
Alright thanks

**Pull request merged**